protocols/dcutr/src/handler/relayed.rs

```diff
@@ -310,14 +304,20 @@ impl ConnectionHandler for Handler {
                     );
                 }
             }
-            Command::UpgradeFinishedDontKeepAlive => {
-                self.keep_alive = KeepAlive::No;
-            }
+            Command::UpgradeFinishedDontKeepAlive => {}
```

Comment:
I think we can just delete this variant now?

Reply:
@thomaseizinger Yes you are right, this is done.